cube/assets/shader.f.glsl

```diff
@@ -0,0 +1,11 @@
+#version 100
```

Comment:
Yep, multiple shaders is definitely a thing. Not sure if you'll need multiple in this example, but it is frequent in even slightly complicated applications. For example, 1 shader pair for rendering the world, one for rendering a 2d ui overlay.